deepfence_agent/binary-install-scripts/start_deepfence_agent.sh

```diff
@@ -0,0 +1,159 @@
+#!/bin/bash
```

Comment:
Do we need really need a new script?
I would rather keep our one start deepfenced script to avoid misses and future bugs